doc/developers/contributing.rst

```diff
@@ -872,6 +872,51 @@ from high-level questions to a more detailed check-list.
   instructive?
 
 
+Standard replies
+................
+
+It may be helpful to store some of these in GitHub's [saved
+replies](https://github.com/settings/replies/) for reviewing:
+
+Issue: Update the docs svp
+    ::
+        Please feel free to offer a pull request updating the documentation if
+        you feel it could be improved.
```

Comment:
Seems a bit strange that we restrict the contribution to documentation. Also, I might prefer a link to the contribution guide here to avoid contributors from making low-level mistakes.

Reply:
I think this is meant for things that are particular to the documentation.

Reply:
This is meant because people come up with issues all the time, where the best solution is to improve the documentation.